backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/descriptors/ClassVtablesBuilder.kt

```diff
@@ -82,26 +87,57 @@ internal class OverriddenFunctionDescriptor(
 }
 
 internal class ClassVtablesBuilder(val classDescriptor: ClassDescriptor, val context: Context) {
+    private val DEBUG = 0
+
+    private inline fun DEBUG_OUTPUT(severity: Int, block: () -> Unit) {
+        if (DEBUG > severity) block()
+    }
+
     val vtableEntries: List<OverriddenFunctionDescriptor> by lazy {
 
         assert(!classDescriptor.isInterface)
 
+        DEBUG_OUTPUT(0) {
+            println()
+            println("BUILDING vTable for ${classDescriptor.descriptor}")
+        }
+
         val superVtableEntries = if (classDescriptor.isSpecialClassWithNoSupertypes()) {
             emptyList()
         } else {
             val superClass = classDescriptor.getSuperClassNotAny() ?: context.ir.symbols.any.owner
             context.getVtableBuilder(superClass).vtableEntries
         }
 
-        val methods = classDescriptor.sortedOverridableOrOverridingMethods
+        // TODO: extract method .isBridge()
+        val methods = classDescriptor.sortedOverridableOrOverridingMethods.filterNot { it.name.asString().contains("<bridge-") }
```

Comment:
Hmm, not very reliable approach.

Reply:
Yeah, and because of this I placed this "TODO", also I has a fix for that in another branch about devirtualized bridges inlining.